### PR TITLE
Homepage task15 Fetch and display meals on homepage Student A

### PR DESCRIPTION
**Pull request by Yordinia**
# Hi @misalazeem How are you?

###  These are the tasks for this branch

- When the page loads, the webapp retrieves data from the selected API and shows the list of items on screen with the basic data (e.g. title + image).

- /Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

- This task does not include displaying the number of likes for each item.

- This task does not include the counter of the items.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+}
+
+.homepage {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 20px;
+  background-color: rgb(206, 203, 203);
+  margin: auto;
+  padding: 3rem;
+}
```
Comment (misalazeem) [inline src/style.css:70]: image

Please make sure you add responsiveness for the mobile version of the app.

**Review by misalazeem — CHANGES_REQUESTED** [review]
Hi @Yordinia,

Good job so far on the Capstone project.
Below are some required changes before we Proceed.
Here are some highligts:
- Project is well structured.
- API is working as per the specification

Following are the required changes.

File: src/style.css (hunk 1 above)
Comment (Yordinia) [reply src/style.css:70]: Hi @misalazeem I have now added the mobile styling to this app
Thanks for the feed back ?!

**Comment by Yordinia** [comment]
@misalazeem 
Thanks for the feed back

**Review by misalazeem — APPROVED** [review]
Well done.
Status: Approved
Merge IT

**Pull request merged**